websites/N/Neon/dist/metadata.json

```
@@ -0,0 +1,41 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Neon",
+    "author": {
+      "name": "Kety The Folf",
+      "id": "263443630767734784"
+    },
+    "contributors": [
+        {
+          "name": "SlowLife",
+          "id": "374905512661221377"
+        }
+      ],
+    "category": "videos",
+    "logo": "https://i.imgur.com/VdcSlNL.png",
+    "thumbnail": "https://i.imgur.com/dO4SNtS.png",
+    "version": "1.0.0",
+    "color": "#000000",
+    "tags": ["videos", "streams"],
+    "url": [
+      "www.neontv.co.nz",
+      "neontv.co.nz"
+    ],
+    "description": {
+      "en": "Neon, marketed as NEON until 2020, is a subscription video on demand service offering a range of television shows and movies. The platform is a division of Sky Network Television Limited."
```

```suggestion
      "en": "Watch hit TV shows on-demand and rent blockbuster movies with NEON. No term contracts."
```
Use the official description.